### ISSUE TITLE
varcheck: unused consts are reported as "global variable"

### ISSUE THREAD

**Issue by dolmen** [alecthomas/gometalinter#259]
`varcheck` reports both unused variables and unused named constants.
But the [`MessageOverride`](../blob/8747b3f8d7af209a05841d9e07ea32be6044e01e/config.go#L153) in gometalinter is only about variables: `unused global variable {message}`

Test case:
```go
package main

const x = 3
```
```
$ gometalinter -D deadcode -E varcheck 
main.go:3:7:warning: unused global variable x (varcheck)
```

Workaround: fix message override from the command line
```
$ gometalinter -D deadcode -E varcheck '--message-overrides=varcheck:unused variable or constant {message}'
main.go:3:7:warning: unused variable or constant x (varcheck)
```

**Comment by alecthomas** [comment]
Thanks, though it probably would have been faster to send me a PR!
